src/common/data/enums/TransactionKind.ts

```diff
@@ -1,6 +1,7 @@
 enum TransactionKind {
-  SEND = 'Send',
+  SEND = 'Sent',
```

Comment:
Is there a reason to convert the Send literal to Send? We kept it as Send because the backend would contain the same literal in its transactionType variable which helps the UI w/ transaction kind recognition.